mkosi/installer/apt.py

```diff
@@ -62,12 +62,12 @@ def scripts(cls, context: Context) -> dict[str, list[PathString]]:
                 "apt-mark",
                 "apt-sortpkgs",
             )
-        } | {
-            "mkosi-install"  : apivfs_cmd(context.root) + cls.cmd(context, "apt-get") + ["install"],
-            "mkosi-upgrade"  : apivfs_cmd(context.root) + cls.cmd(context, "apt-get") + ["upgrade"],
-            "mkosi-remove"   : apivfs_cmd(context.root) + cls.cmd(context, "apt-get") + ["purge"],
-            "mkosi-reinstall": apivfs_cmd(context.root) + cls.cmd(context, "apt-get") + ["install", "--reinstall"],
-        }
+        } | cast(dict[str, list[PathString]], {
```

Comment:
Can we get a todo here noting that this is another workaround for pyright?

Reply:
Will do a follow up